src/courtroom.cpp

```diff
@@ -2763,6 +2780,10 @@ void Courtroom::start_chat_ticking()
     this->do_flash();
     sfx_player->play(ao_app->get_custom_realization(m_chatmessage[CHAR_NAME]));
   }
+  int emote_mod = m_chatmessage[EMOTE_MOD].toInt(); // text meme bonanza
```

Comment:
why was this moved?

Reply:
Consistency, variable not necessary for log parsing until the spot it was moved to

Reply:
Wait oh fuckin it's because if chatmessage is empty then the screenshake won't fire without moving this @scatterflower